CHANGELOG.md

```diff
@@ -13,6 +13,17 @@
 [All changes in [[UnreleasedVersion]]](https://github.com/mozilla/uniffi-rs/compare/v0.12.0...HEAD).
 
 ### ⚠️ Breaking Changes ⚠️
+- The Swift bindings now explicitly generate two separate Swift modules, one for
+  the high-level Swift code and one for the low-level C FFI. This change is intended
+  to simplify distribution of the bindings via Swift packages, but brings with it
+  some changes to the generated file layout.
+  - For an interface namespace "example", we now generate:
+    - A bridged C module named "exampleFFI" containing the low-level C FFI,
+      consisting of an `exampleFFI.h` file and matching `exampleFFI.modulemap`
+      file. The name can be customized using the `ffi_module_name` config option.
```

Comment:
FWIW, I'm open to the suggestion that we should leave the default names of these files unchanged in order to avoid disruption to consumers, I just think the new names may help form a more clear mental model of what's happening.